ruby/pure-ruby/config.reek

```diff
@@ -5,3 +5,7 @@ Attribute:
 
 TooManyStatements:
   enabled: false
+
+UtilityFunction:
+  public_methods_only: false
```

Comment:
I generally like this, since I found that `public_methods_only: true` did not function sensibly with the `private def method ...` way of declaring private methods.